influxql/select.go

```diff
@@ -279,7 +279,7 @@ func buildExprIterator(expr Expr, ic IteratorCreator, opt IteratorOptions, selec
 			opt.Interval = Interval{}
 
 			return newHoltWintersIterator(input, opt, int(h.Val), int(m.Val), includeFitData, interval)
-		case "derivative", "non_negative_derivative", "difference", "moving_average", "elapsed":
+		case "derivative", "non_negative_derivative", "difference", "moving_average", "elapsed", "integral":
```

Comment:
`integral()` seems to be different than the rest of these. The rest of these are here because they aren't aggregates so they don't support `GROUP BY` or `fill()`. If this is going to be an aggregate, it needs to be moved to the area below this.